docker-compose.yml

```diff
@@ -122,6 +122,45 @@ services:
     labels:
       logs: "promtail"
 
+### Web Client
+  hyperswitch-web:
```

Comment:
also can we add 
```yaml
    labels:
      logs: "promtail"
```

here to allow promtail to scrape these logs?

Reply:
Sure